### DISCUSSION TITLE
trying to connect to multical 21 via optical connection

### DISCUSSION THREAD

**Discussion by oliverjoa** [wmbusmeters/wmbusmeters#935] [General]
Hi,

I am trying to get wmbusmeters connect to my multical 21 via the optical ir connection.
Anyone out there who did that already?
I hoped to get it working with a simple "ir to uart converter". 
Tried with:
./wmbusmeters /dev/ttyUSB0:mbus:2400 MyTapWater multical21:c1 63594301 NOKEY
but nothing happens.
Any Idea?

**Comment by weetmuts** [comment]
I don't think the optical ir offers an mbus protocol....I think it is a proprietary protocol called KMP?
